generators/app/index.js

```diff
@@ -91,19 +124,68 @@ module.exports = yeoman.Base.extend({
         type: 'input',
         name: constants.PROP_PROJECT_GROUP_ID,
         message: 'Project groupId?',
-        default: this.config.get(constants.PROP_PROJECT_GROUP_ID),
+        default: this._getConfigValue(constants.PROP_PROJECT_GROUP_ID),
+        when: function(props) {
+          if(this.bail) return false;
```

Comment:
To be clear, I'm on the fence about if the original code or my suggestion is better :)